xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -388,9 +388,18 @@ The type of signing key represented by <paramref name="issuerCertificate" /> cou
 
  -or-
 
-<paramref name="issuerCertificate" /> has a different key algorithm than the requested certificate.</exception>
+<paramref name="issuerCertificate" /> has a different key algorithm than the requested certificate.
+
+-or-
+
+The <see cref="P:System.Security.Cryptography.X509Certificates.X509Certificate2.HasPrivateKey" /> value for <paramref name="issuerCertificate" /> is <see langword="false" />.</exception>
         <exception cref="T:System.InvalidOperationException">
-          <paramref name="issuerCertificate" /> is an RSA certificate and the current object was created using a constructor that doesn't accept a <paramref name="padding" /> parameter.</exception>
+          <paramref name="issuerCertificate" /> is an RSA certificate and the current object was created using a constructor that doesn't accept a <paramref name="padding" /> parameter.
+
+-or-
+
+<paramref name="issuerCertificate" /> is an RSA certificate and this object was created via a constructor
+              which does not accept a <see cref="T:System.Security.Cryptography.RSASignaturePadding" /> value.</exception>
```

Comment:
```suggestion
              that doesn't accept a <see cref="T:System.Security.Cryptography.RSASignaturePadding" /> value.</exception>
```

Reply:
I'll apply your suggestion, although it bugs me that the sentence is split into two lines like that, so I'll apply it in a separate two-line suggestion.